src/main/java/com/alkemy/ong/service/impl/CommentaryServiceImpl.java

```diff
@@ -92,4 +96,11 @@ public void deleteById(Long id, HttpServletRequest request) {
             throw new ForbiddenException("User with no access.");
         }
     }
+
+    public List<CommentaryBodyDTO> findAllById(Long id){
+        News news = newsRepository.findById(id).orElseThrow(() -> new NotFoundException("Post no exist Id: "+id));
+        List<Commentary> listAll = commentaryRepository.findByNews(news);
```

Comment:
Lo unico que le agregaria es un mensaje si no tiene comentarios. Para que muestre eso en lugar de un body vacio

Reply:
mmmm no, no deberia mostrar un 404, no te olvides que el codigo es universal y un 404 es un not found, vos aca si encontraste algo solo que lo que encontras es vacio, deberia ser un codigo 200 porque la peticion esta OK salvo que no hay contenido. En todo caso podria ser un not content, pero con que haya un 200 esta bien.

Reply:
Claro por eso te decía que use un  NotFoundException . Me parecía que estaba mal usar el 404 , ahí lo modifico .

Reply:
Turco Como hablamos modifique en el controller para chequear si la lista está vacía y mandar un ok con el mensaje de lista vacía.
![image](https://user-images.githubusercontent.com/54463198/175438870-828d20e1-fe9b-4e69-9ea0-72d583e1263b.png)